### PR TITLE
Disable parent on the fly for tags

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #39441

### Summary of Changes
This PR disables the input of a new tag in the parent field on com_tags edit. 

### Testing Instructions
see #39441 
Ty to add a new tag in the parent selection field.

### Actual result BEFORE applying this Pull Request
see #39441 

### Expected result AFTER applying this Pull Request
The parent tag allows seleting a tag as parent but does not accept the input of a new tag name. 

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
Not sure this is quite correct.

If you look at the category selector in com_categories and enter a non-existant category as the parent you can not - pressing enter that this point does nothing
image

But even with this pr when you do the same in tags then pressing enter will appear to create the tag except its not saved.
image

**Comment by chmst** [comment]
Agree, The tag field should behave like the categoryedit field. At the moment it os not clear how the categoryedit field should work see https://github.com/joomla/joomla-cms/issues/39440.
This is a quick fix to resolve the ugly error message, 

I don't mind to close this one and re-open the issue. Unfortunately don't have time for diving deeper into nested set.

File: administrator/components/com_tags/forms/tag.xml [hunk 1]
```diff
@@ -23,10 +23,6 @@
 		name="parent_id"
 		type="tag"
 		label="COM_TAGS_FIELD_PARENT_LABEL"
-		mode="nested"
-		validate="notequals"
-		field="id"
-		parent="parent"
```
Comment (joomdonation) [inline administrator/components/com_tags/forms/tag.xml]: Could you please revert the change you made here and add `custom="false"` instead ? It should address the issue you are trying to solve if I understand it correctly.

Comment (chmst) [reply administrator/components/com_tags/forms/tag.xml]: thanks, done

**Comment by joomdonation** [comment]
I have not tested this item.<br /><br />Works as described.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39443">issues.joomla.org/tracker/joomla-cms/39443</a>.</sub>

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on 922e402f79e479ae7a46c04cb694d9497f18560b<br /><br />Works as described.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39443">issues.joomla.org/tracker/joomla-cms/39443</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 922e402f79e479ae7a46c04cb694d9497f18560b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39443">issues.joomla.org/tracker/joomla-cms/39443</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39443">issues.joomla.org/tracker/joomla-cms/39443</a>.</sub>

**Comment by fancyFranci** [comment]
Great, thank you!
